src/utils/storage.spec.ts

```diff
@@ -0,0 +1,30 @@
+import Storage from './storage';
+
+interface Person {
+  name: string
+}
+
+describe('Storage', () => {
+  let storage: Storage<Person>;
+  beforeEach(() => {
+    storage = new Storage<Person>();
+  });
+  it('should add an element in the storage', () => {
+    const saved = storage.save({ name: 'santiago' });
+    expect(storage.get(saved)).toEqual({ name: 'santiago' });
+  });
+
+  it('should remove from storage', () => {
+    const toAdd = ['marcos', 'thiago'];
+    toAdd.forEach((element) => storage.save({ name: element }));
```

Comment:
this function in the forEach is the same you use in the next test, put this in a separated function